### PR TITLE
feat: 🎸 add metrics for all the volumes

### LINKED ISSUE #1561: Check the disk usage of all the storages in metrics
See https://www.notion.so/huggingface2/Disk-storage-0a4a8fcf27754c8cb7248b259dcc4b21 (internal)

### PR DESCRIPTION
**Pull request by severo**
fixes #1561

### CODE REVIEW

File: chart/templates/services/admin/_container.tpl [hunk 1]
```diff
@@ -12,6 +12,10 @@
   {{ include "envCommon" . | nindent 2 }}
   {{ include "envLog" . | nindent 2 }}
   {{ include "envMetrics" . | nindent 2 }}
+  {{ include "envParquetMetadata" . | nindent 2 }}
+  {{ include "envWorker" . | nindent 2 }}
```
Comment (AndreaFrancis) [inline chart/templates/services/admin/_container.tpl:16]: Not sure if we should include all the worker configs here, maybe just the ones that are needed? (I think it is DUCKDB_INDEX_CACHE_DIRECTORY and DESCRIPTIVE_STATISTICS_CACHE_DIRECTORY)

File: tools/docker-compose-datasets-server.yml [hunk 2]
```diff
@@ -28,6 +28,10 @@ services:
     build:
       context: ..
       dockerfile: services/admin/Dockerfile
+    volumes:
+      - assets:${ASSETS_STORAGE_DIRECTORY-/assets}:rw
+      - duckdb-index:${DUCKDB_INDEX_CACHE_DIRECTORY-/duckdb-index}:rw
```
Comment (AndreaFrancis) [inline tools/docker-compose-datasets-server.yml:33]: Do you know if the volume for statistics is missing here? Same for changes on line 52-55

File: tools/docker-compose-datasets-server.yml [hunk 3]
```diff
@@ -28,6 +28,10 @@ services:
     build:
       context: ..
       dockerfile: services/admin/Dockerfile
+    volumes:
+      - assets:${ASSETS_STORAGE_DIRECTORY-/assets}:rw
```
Comment (AndreaFrancis) [inline tools/docker-compose-datasets-server.yml:32]: Why not just read for these volumes?

File: libs/libcommon/src/libcommon/prometheus.py [hunk 4]
```diff
@@ -76,15 +100,35 @@ def update_responses_in_cache_total() -> None:
         ).set(cache_metric.total)
 
 
-def update_assets_disk_usage(assets_directory: StrPath) -> None:
+def update_disk_gauge(gauge: Gauge, directory: StrPath) -> None:
     # TODO: move to metrics, as for the other metrics (queue, cache)
-    total, used, free, percent = disk_usage(str(assets_directory))
+    total, used, free, percent = disk_usage(str(directory))
```
Comment (AndreaFrancis) [inline libs/libcommon/src/libcommon/prometheus.py:105]: In the next line, it should update the labels for the new gauge var, not for ASSETS_DISK_USAGE, I think this is the problem.

File: chart/templates/services/admin/_container.tpl (hunk 1 above)
Comment (severo) [reply chart/templates/services/admin/_container.tpl:16]: Yes, we might be more precise. I'll change it, also: it will be clearer in the admin template

**Pull request merged**